src/NJsonSchema.CodeGeneration/ValueGeneratorBase.cs

```diff
@@ -51,6 +64,15 @@ public virtual string GetDefaultValue(JsonSchema4 schema, bool allowsNull, strin
             return null;
         }
 
+        private string GetStringValue(JsonObjectType type, object value, string format)
+        {
+            if(_fomatNotCompatibileWithString.Contains(format))
+                return null;
+            else
+                return "\"" + ConversionUtilities.ConvertToStringLiteral(value.ToString()) + "\"";
+            //TODO : eg. DateTime.Prase("\"" + ConversionUtilities.ConvertToStringLiteral(value.ToString()) + "\""); for c#
```

Comment:
Please create a new issue for this todo and remove the todo

Reply:
Done